deploy/helm-charts/server/values.yaml

```diff
@@ -8,3 +8,22 @@ name: terrascan
 persistence:
   enabled: false
   existingclaim: terrascanPvc
+
+webhook:
+  name: webhook.terrascan.io
+  failurePolicy: Ignore
```

Comment:
Had to change this to `Ignore`, because if service is un-available for webhook, it should block all the specified resources in cluster.
Hope this is the desired behaviour.
cc: @dev-gaur @kanchwala-yusuf @jlk